frontend/assets/javascripts/src/modules/form/payment/displayCardImg.js

```diff
@@ -0,0 +1,19 @@
+define(['stripe'], function (stripe) {
+
+    var CARD_SPRITE_ICON_PREFIX = 'sprite-card--';
+    var CREDIT_CARD_IMG_ELEM = document.querySelector('.js-credit-card-image');
+
+    /**
+     * display the credit card image within the credit card input dependant on the stripe lib cardType helper response
+     * @param cardNumber
+     */
+    var displayCardImg = function (cardNumber) {
+        var cardType = stripe.cardType(cardNumber).toLowerCase().replace(' ', '-');
+        var regEx = new RegExp('\\b' + CARD_SPRITE_ICON_PREFIX + '\\S+', 'g');
```

Comment:
Not sure if it's worth pulling the regEx definition outside the function so it's not redefined everytime the function is called?